static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -0,0 +1,296 @@
+          <ErrorBoundary mini>
+            <TextCopyInput size="sm">{feedbackItem.url}</TextCopyInput>
+          </ErrorBoundary>
+        </Section>
+
+        {feedbackItem.replay_id ? (
+          <ReplaySection organization={organization} replayId={feedbackItem.replay_id} />
+        ) : null}
+
+        <TagsSection tags={feedbackItem.tags} />
+
+        <Section icon={<IconJson size="xs" />} title={t('Raw')}>
+          <ObjectInspector
+            data={feedbackItem}
+            expandLevel={3}
+            theme={{
+              TREENODE_FONT_SIZE: '0.7rem',
+              ARROW_FONT_SIZE: '0.5rem',
+            }}
+          />
+        </Section>
+      </OverflowPanelItem>
+    </FeedbackItemContainer>
+  );
+}
+
+const FeedbackItemContainer = styled(FluidHeight)`
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+`;
+
+const HeaderPanelItem = styled(PanelItem)`
+  display: grid;
+  padding: ${space(1)} ${space(2)};
+`;
+
+const OverflowPanelItem = styled(PanelItem)`
+  overflow: scroll;
+
+  flex-direction: column;
+  gap: ${space(3)};
+`;
+
+function Username({feedbackItem}: {feedbackItem: HydratedFeedbackItem}) {
+  const displayValue = feedbackItem.user.display_name || feedbackItem.contact_email;
+  const hasBoth = feedbackItem.user.display_name && feedbackItem.contact_email;
+  if (!displayValue) {
+    <strong>{t('Unknown User')}</strong>;
+  }
+
+  const Purple = styled('span')`
+    color: ${p => p.theme.purple300};
+  `;
+
+  return (
+    <Flex gap={space(1)} align="center">
+      <strong>
+        {hasBoth ? (
+          <Fragment>
+            {feedbackItem.user.display_name}
+            <Purple>•</Purple>
+            {feedbackItem.contact_email}
+          </Fragment>
+        ) : (
+          displayValue
+        )}
+      </strong>
+      {feedbackItem.contact_email ? (
+        <CopyToClipboardButton
+          size="xs"
+          iconSize="xs"
+          text={feedbackItem.contact_email}
+        />
+      ) : null}
+    </Flex>
+  );
+}
+
+function Viewers({feedbackItem: _}: {feedbackItem: HydratedFeedbackItem}) {
+  const displayUsers = [
+    {
+      email: 'colton.allen@sentry.io',
+      id: '1',
+      ip_address: '',
+      name: 'Colton Allen',
+      username: 'cmanallen',
+    },
+  ];
+
+  return (
+    <AvatarList
+      users={displayUsers}
+      avatarSize={28}
+      maxVisibleAvatars={13}
+      renderTooltip={user => (
+        <Fragment>
+          {userDisplayName(user)}
+          <br />
+          <DateTime date={(user as AvatarUser).lastSeen} />
+        </Fragment>
+      )}
+    />
+  );
+}
+
+function ResolveButton({feedbackItem}: {feedbackItem: HydratedFeedbackItem}) {
+  if (feedbackItem.status === 'unresolved') {
+    return (
+      <Button priority="primary" size="xs" icon={<IconArchive />}>
+        {t('Resolve')}
+      </Button>
+    );
+  }
+
+  return (
+    <Button size="xs" icon={<IconArchive />}>
+      {t('Un-Resolve')}
```

Comment:
We don't hyphenate this  with issues

![image](https://github.com/getsentry/sentry/assets/79684/f576956e-53eb-4ed8-9bd0-3d954028fce4)

Reply:
fixed.